validator.py

```diff
@@ -571,28 +571,29 @@ def validate_quantized_support(self):
 
     def validate_quantized_support_in_characters(self, characters):
         for character in characters:
-            if 'injuries' in character:
-                for injury in character['injuries']:
-                    if 'treatments_required' in injury:
-                        required = injury['treatments_required']
-                        type = injury['name']
-                        location = injury['location']
-                        # look for an injury type that doesn't support quantized injuries
-                        if required > 1 and not self.supports_quantized_injury(type, location):
-                            self.logger.log(LogLevel.ERROR, f"Injuries requiring multiple treatments are only supported when the injury is treated by hemostatic gauze or pressure bandage, but not '{type}' injuries at '{location}' location in character '{character['id']}'.")
-                            self.invalid_values += 1
+            for injury in character.get('injuries', []):
+                if 'treatments_required' in injury:
+                    required = injury['treatments_required']
+                    type = injury['name']
+                    location = injury['location']
+                    # look for an injury type that doesn't support quantized injuries
+                    if required > 1 and not self.supports_quantized_injury(type, location):
+                        self.logger.log(LogLevel.ERROR, f"Injuries requiring multiple treatments are only supported when the injury is treated by hemostatic gauze or pressure bandage, but not '{type}' injuries at '{location}' location in character '{character['id']}'.")
+                        self.invalid_values += 1
 
 
     def supports_quantized_injury(self, injury_type, location):
         '''
         Returns whether the specified injury/location combination is successfully treated by a hemostatic gauze or pressure bandage.
         '''
         if injury_type == 'Laceration' and 'thigh' in location:
-            return False
-        if injury_type == 'Shrapnel' and 'face' not in location:
-            return False
+            return False # takes a Tourniquet
+        if injury_type == 'Shrapnel' and 'face' in location:
+            return False # untreatable
```

Comment:
face shrapnel is treatable with nasal airway

Reply:
Duh...fixed.